github action to remind the Office Hours session in Slack

Adity20 opened the pull request:
## Issue solved
This pull request adds a new GitHub Action that sends reminders to join the Office Hours session on the Friday before the session and on the same day of the session.

## summary
The GitHub Action is scheduled to run at 15:00 every Friday and at 9:00 every Tuesday. It uses the `@octokit/core` package to fetch the latest issue with the `OCWM_LABEL` label from the repository. It then sends a message to the Slack channel with the issue link and date.

benjagm commented on .github/workflows/ocwm-reminders-friday-tuesday.yml:
```diff
@@ -0,0 +1,64 @@
+
+on:
+  schedule:
+    - cron: '0 15 * * 5'  # Runs every Friday at 15:00
```
This is running every week bet the meeting is monthly. This should run the friday before and the same day of the meeting.

Adity20 replied:
The cron jobs will now run as follows 
cron: '0 15 1-7 * 5' #Runs at 15:00, between day 1 and 7 of the month on Friday 
    - cron: '0 15 1-7 * 2' #Runs at 15:00, between day 1 and 7 of the month on Tuesday

benjagm commented on .github/workflows/ocwm-reminders-friday-tuesday.yml:
```diff
@@ -0,0 +1,64 @@
+
+          const issueNumber = workMeetings[0].number
+          const newTitle = workMeetings[0].title;
+          const issueDate = newTitle.replace(/JSON Schema office hours /g, "");
```
I think we need to calculate the exact date of the Office Hours and pass it to slack

Adity20 replied:
Added the logic for this in new commits to the code.

benjagm reviewed (changes requested):
Leave some comments. Thanks!!

Adity20 commented:
changes made as per the request @benjagm.
If any more changes are required let me know :)

Adity20 commented on .github/workflows/ocwm-reminders-tuesday.yml:
```diff
@@ -0,0 +1,78 @@
+  schedule:
+    - cron: '0 15 1-7 * 5' #Runs at 15:00, between day 1 and 7 of the month on Friday 
+    - cron: '0 15 1-7 * 2' #Runs at 15:00, between day 1 and 7 of the month on Tuesday
+
```
Here an issue might occur that Tuesday comes first before the Friday in upcoming month I have handled that below if the logic seems good to you tell me otherwise I can make appropriate changes in that

Adity20 commented:
I made the requested changes @benjagm.
If you are not able to see them let me know

benjagm commented:
> I made the requested changes @benjagm. If you are not able to see them let me know

Ok. I left many comments and I don't see any reply, reaction or action so wasn't sure about the status.

Adity20 commented:
Maybe due to adding a new file deleting the last one

benjagm commented:
Can you please go through the comments I left in my last review and reply/resolve them?

Adity20 commented:
I resolved the issues you asked me to @benjagm.
I created another file with addressing the issues
1. Now the reminder will be sent friday before and the same day.
2. Passing the exact date to slack and same day value as today.

benjagm commented:
Can you comment or resolve my previous review comments please?
